registry.json

```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"ONE1",
+      "poolId":"pool1n2yp66zk6m33sg8k20hsvmk9tw7xeeswzdcfxhnx9hm9jkxv5k4",
```

Comment:
Your poolid is : 9a881d6856d6e31820f653ef066ec55bbc6ce60e1370935e662df659

Reply:
```suggestion
      "poolId":"9a881d6856d6e31820f653ef066ec55bbc6ce60e1370935e662df659",
```